code/modules/clothing/shoes/jobs.dm

```diff
@@ -28,12 +28,18 @@
 	icon_state = "kneeboots"
 	item_state = "kneeboots"
 
+/obj/item/clothing/shoes/jackboots/knee/handle_movement(var/turf/walking, var/running)
+	trip_up()
+
 /obj/item/clothing/shoes/jackboots/thigh
 	name = "thigh-length black boots"
 	desc = "Even taller synthleather boots with an artificial shine."
 	icon_state = "thighboots"
 	item_state = "thighboots"
 
+/obj/item/clothing/shoes/jackboots/thigh/handle_movement(var/turf/walking, var/running)
+	trip_up()
```

Comment:
Ditto. Not heels.

Reply:
yeah nah went and looked at a google search and its all heels

also having boots laced up to your thighs is probably a bad idea to run in